README.md

```diff
@@ -1,163 +1,216 @@
-    * [Gitter](#gitter)
-* [DeepChem Publications](#deepchem-publications)
-* [Examples](/examples)
-* [About Us](#about-us)
-* [Citing DeepChem](#citing-deepchem)
+- [Requirements](#requirements)
+- [Installation](#installation)
+  - [Install latest package with conda](#install-via-conda-recommendation)
+  - [Install latest package with pip (WIP)](#install-via-pip-wip)
+  - [Install from source](#install-from-source)
+    - [General installation](#general-installation)
+    - [Use powershell (Windows)](#use-powershell-windows)
+  - [Install using a Docker (WIP)](#install-using-a-docker-with-gpu-wip)
+- [FAQ and Troubleshooting](#faq-and-troubleshooting)
+- [Getting Started](#getting-started)
+- [Contributing to DeepChem](/CONTRIBUTING.md)
+  - [Code Style Guidelines](/CONTRIBUTING.md#code-style-guidelines)
+  - [Documentation Style Guidelines](/CONTRIBUTING.md#documentation-style-guidelines)
+  - [Gitter](#gitter)
+- [DeepChem Publications](#deepchem-publications)
+- [Examples](/examples)
+- [About Us](#about-us)
+- [Citing DeepChem](#citing-deepchem)
 
 ## Requirements
-* [pandas](http://pandas.pydata.org/)
-* [joblib](https://pypi.python.org/pypi/joblib)
-* [sklearn](https://github.com/scikit-learn/scikit-learn.git)
-* [numpy](https://store.continuum.io/cshop/anaconda/)
-* [tensorflow](https://www.tensorflow.org/)
+
+DeepChem requires these packages on any condition.
+
+- [joblib](https://pypi.python.org/pypi/joblib)
+- [pandas](http://pandas.pydata.org/)
+- [scikit-learn](https://github.com/scikit-learn/scikit-learn.git)
+- [tensorflow](https://www.tensorflow.org/)
+  - `deepchem>=2.4.0` requires tensorflow v2
+  - `deepchem<2.4.0` requires tensorflow v1
 
 ### Soft Requirements
-DeepChem has a number of "soft" requirements. These are packages which are needed for various submodules of DeepChem but not for the package as a whole.
 
-* [RDKit](http://www.rdkit.org/docs/Install.html)
-* [six](https://pypi.python.org/pypi/six)
-* [MDTraj](http://mdtraj.org/)
-* [PDBFixer](https://github.com/pandegroup/pdbfixer)
-* [Pillow](https://pillow.readthedocs.io/en/stable/)
+DeepChem has a number of "soft" requirements. These are packages which are needed for various submodules of DeepChem but not for the package as a whole.
 
-### Super easy install via pip3
+- [MDTraj](http://mdtraj.org/)
+- [NetworkX](https://networkx.github.io/documentation/stable/index.html)
+- [PDBFixer](https://github.com/pandegroup/pdbfixer)
+- [Pillow](https://pypi.org/project/Pillow/)
+- [RDKit](http://www.rdkit.org/docs/Install.html)
+- [simdna](https://github.com/kundajelab/simdna)
+- [XGBoost](https://xgboost.readthedocs.io/en/latest/)
 
-```bash
-pip3 install joblib pandas sklearn tensorflow pillow deepchem
-```
+## Installation
 
-### Easy Install via Conda
+### Install via conda (Recommendation)
 
 ```bash
-conda install -c deepchem -c rdkit -c conda-forge -c omnia deepchem=2.3.0
+conda install -c deepchem -c rdkit -c conda-forge -c omnia deepchem

```

Comment:
Is it needed to install tensorflow before this command?
From https://github.com/deepchem/deepchem/issues/1880, https://github.com/conda-forge/deepchem-feedstock/pull/2, I seem that the latest cpu package already don't install tensorflow...